src/builder-git.c

```diff
@@ -148,6 +148,31 @@ git_has_version (int major,
   return TRUE;
 }
 
+static gboolean
+git_lfs_is_available (void)
```

Comment:
Would make more sense to move this to the first commit and squash the checks below into the commit that actually calls git lfs. Then every commit is kept in working condition.

Reply:
Can add the CI bits in the first commit, then add the new --disable-lfs option in the next commit but not hook it up yet, then either add the `builder_git_use_lfs` in one commit or merge it with the rest of the changes.